docs/compass/04-01-compass.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Enable Compass in Kyma
+type: Installation
+---
+
+To enable Compass in Kyma, follow the instructions for the [custom component installation](/root/kyma#configuration-custom-component-installation) and enable the `compass` and `compass-runtime-agent` modules.
+You can also [install Kyma on a cluster](/root/kyma#installation-install-kyma-on-a-cluster) with the ready-to-use configurations for different Compass modes.
+
+## Default Kyma installation
+
+This is a preconfigured single-tenant and single-Runtime mode which will eventually become part of the default Kyma installation. It provides the complete cluster Kyma installation with all components, including both Compass and Agent. To enable this mode, follow the cluster Kyma installation and use the [`installer-cr-cluster-with-compass.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-with-compass.yaml.tpl) configuration file.
+
+## Kyma with Compass only
```

Comment:
```suggestion
## Kyma only with Compass
```

Reply:
ref -> [link](https://github.com/YaaS/REST_API_Documentation_Guidelines/wiki/009-Only-the-Lonely#using-only-in-technical-writing)

Reply:
but perhaps the heading could be improved somehow 🤔